components/prism-purescript.js

```diff
@@ -1,5 +1,5 @@
 Prism.languages.purescript = Prism.languages.extend('haskell', {
-	'keyword': /\b(?:ado|case|class|data|derive|do|else|forall|if|in|infixl|infixr|instance|let|module|newtype|of|primitive|then|type|where)\b/,
+	'keyword': /\b(?:ado|case|class|data|derive|do|else|forall|if|in|infixl|infixr|instance|let|module|newtype|of|primitive|then|type|where|∀)\b/,
```

Comment:
The problem is the `\b`s. They are necessary for ASCII keywords but won't work for ∀. `\b` means "asserts a word boundary". Since ∀ isn't a word character, the `\b`s require ∀ to be preceded and followed by a word character (e.g. `a∀a`).


```suggestion
	'keyword': /\b(?:ado|case|class|data|derive|do|else|forall|if|in|infixl|infixr|instance|let|module|newtype|of|primitive|then|type|where)\b|∀/,
```